acceptance/bundle/templates/lakeflow-pipelines/python/output/my_lakeflow_pipelines/resources/lakeflow_pipelines_etl/transformations/sample_trips_my_lakeflow_pipelines.py

```diff
@@ -10,4 +10,6 @@
 
 @dlt.table
 def sample_trips_my_lakeflow_pipelines():
-    return spark.read.table("samples.nyctaxi.trips").withColumn("trip_distance_km", utils.distance_km(col("trip_distance")))
+    return spark.read.table("samples.nyctaxi.trips").withColumn(
```

Comment:
this looks like a different formatting output, which could conflict with the in-workspace formatter.